yarn.lock

```diff
@@ -4773,29 +4773,29 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@sentry-internal/tracing@npm:7.54.0":
-  version: 7.54.0
-  resolution: "@sentry-internal/tracing@npm:7.54.0"
+"@sentry-internal/tracing@npm:7.56.0":
```

Comment:
I'm a bit concerned by all these other package upgrades. Maybe it's ok?

Reply:
This is odd indeed. I would expect only Social Logos related changes, like last time we bumped the version in c4a4c445ead180a57c114b6089536fb336d39ffe.

I force-pushed a9fc5f38c35eafbeea52ea3599a04792d6481677 to this branch to try to fix this.